doc/tutorials/introduction/windows_install/windows_install.markdown

```diff
@@ -318,7 +318,8 @@ libraries). If you do not need the support for some of these, you can just freel
     -   *BUILD_PERF_TESTS* -\> There are also performance tests for many OpenCV functions. If
         you are concerned about performance, build them and run.
     -   *BUILD_opencv_python* -\> Self-explanatory. Create the binaries to use OpenCV from the
-        Python language.
+        Python language. 
+    -   *BUILD_opencv_world* -\> Builds OpenCV module as one large "opencv_world" binary.With this your OpenCV download can be built into one large .lib or .dll file.
```

Comment:
The second sentence is not necessary, it gives same information as the first one.